<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gscalbiInterfacePrincipal</class>
 <widget class="QDialog" name="gscalbiInterfacePrincipal">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>625</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>625</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>GSCALBI v1.0</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>460</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbxElementos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Elementos a analizar</string>
   </property>
   <widget class="QLabel" name="lblElementosCapa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capa</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbElementosCapa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblElementosClusters">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad de cl√∫sters</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnElementosClusteres">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QListView" name="lvwElementosVariables">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>51</red>
          <green>153</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="HighlightedText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>51</red>
          <green>153</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="HighlightedText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>51</red>
          <green>153</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="HighlightedText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblElementosVariables">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Variables de la capa 'X'&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbIdElemento">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblElementosClusters_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Id de la capa</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbxLocaciones">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>211</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Locaciones a tener en cuenta</string>
   </property>
   <widget class="QLabel" name="lblLocacionesCapa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capa</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbLocacionesCapa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnLocacionesClusteres">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QListView" name="lvwLocacionesVariables">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>51</red>
          <green>153</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="HighlightedText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>48</red>
          <green>144</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="HighlightedText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>51</red>
          <green>153</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="HighlightedText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblLocacionVariables">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>201</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Variables de la capa 'X'&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLocacionClusters">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad de cl√∫sters</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbIdLocaciones">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblElementosClusters_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Id de la capa</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnAbout">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Acerca de GSCALBI</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbxPosicion">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Posici√≥n en el espacio</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QDoubleSpinBox" name="dspnDistancia">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="maximum">
     <double>999999999.999999046325684</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lblDistancia">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Distancia (Km)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPosicion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>&lt;&lt; Sin especificar &gt;&gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Norte</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Este</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oeste</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblPosicion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Posici√≥n</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbxDireccion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Carpeta de Destino</string>
   </property>
   <widget class="QPushButton" name="btnDireccion">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>19</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDireccion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbxAsociacion">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Relaci√≥n</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QComboBox" name="cmbAsociacion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnNuevoCampo">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>A√±adir nuevo campo a capa</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbMetodoAgrupacion">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>SimpleKMeans</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HierarchicalClusterer</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>M√©todo de agrupamiento</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cmbElementosCapa</tabstop>
  <tabstop>spnElementosClusteres</tabstop>
  <tabstop>lvwElementosVariables</tabstop>
  <tabstop>cmbLocacionesCapa</tabstop>
  <tabstop>spnLocacionesClusteres</tabstop>
  <tabstop>lvwLocacionesVariables</tabstop>
  <tabstop>btnAceptar</tabstop>
  <tabstop>btnCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
